crates/core/src/client/config.rs

```diff
@@ -22,6 +22,25 @@ pub enum Network {
     ProviderUrl(Url),
 }
 
+impl Network {
+    pub fn get_gateway_url(&self) -> Result<Url, ConfigError> {
+        match self {
+            Network::Mainnet => Ok(Url::parse("https://alpha-mainnet.starknet.io/feeder_gateway/").unwrap()),
```

Comment:
can we avoid unwrap() here? in case one day we change this URL

Reply:
We could, you prefer that we return a config error? I usually just unwrap when I know the unwrap is safe

Reply:
I think replacing with a "?" would work here no?